src/content/blog/2024_zne_catalyst.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Just in time for just-in-time error mitigation: Mitiq meets Catalyst"
+author: Alessandro Cosentino
+day: 18
+month: 11
+year: 2024
+tags:
+- Mitiq
+- Partnership
+---
+
+This blog post is a summary of a partnership between Unitary Fund and 
+[Xanadu](https://www.xanadu.ai/) to integrate quantum error mitigation techniques
+from Unitary Fund's Mitiq into Xanadu's PennyLane Catalyst compiler.
+
+## Just-in-time compilation
+
+We all want faster computer programs. Just-in-time (JIT) compilation has been a key concept in classical compiler design to help achieve faster computer programs. The basic idea behind a JIT compiler is that it observes the program running and optimizes it based on those observations. Most modern programming languages and runtime environments nowadays support some kind of JIT optimization techniques. In fact, very recently, an [experimental JIT compiler](https://docs.python.org/3/whatsnew/3.13.html#whatsnew313-jit-compiler) was added to Python in its latest 3.13 release.
+
+We all want faster quantum computer programs. Just as JIT compilation accelerates classical programs, similar techniques can be applied to quantum computing for boosting performance.   [Catalyst](https://github.com/PennyLaneAI/catalyst) is a JIT compiler framework developed by Xanadu to speed up quantum programs written with their PennyLane library. If you have a PennyLane program, you can easily decorate it with some simple Catalyst syntax and, in many cases, you have a faster quantum computer program.
```

Comment:
@cosenal something that is missing from the blog post (but is still interesting/novel) is the fact that the new ZNE transformations _work with structure_ -- that is, they aren't unrolling loops; could be worth adding here, because this means that you can apply error mitigation to very compact program representations.

Reply:
great suggestion – added in [596b1a6](https://github.com/unitaryfund/unitary.fund/pull/604/commits/596b1a67b23a41aec19f549fc2d7e99da7440f9d)